CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   The `veecle-telemetry` feature flag has been removed from `veecle-os-runtime`.
   Use the `telemetry-enable` feature flag on `veecle-os` to control telemetry behavior.
 * Added `ThreadAbstraction` trait to OSAL for querying current thread id.
+* Updated MSRV to 1.90.
```

Comment:
Shouldn't this be marked as breaking?

Reply:
Makes sense until we stabilize our API. I think you already created an issue to track Rust version bumps closely. Could you please also create one to add this to the readme/contributing? I cannot access Linear atm.